pkg/plugins/backendplugin/client.go

```diff
@@ -41,17 +41,42 @@ func newClientConfig(executablePath string, logger log.Logger, versionedPlugins
 	}
 }
 
+// LegacyCallbackFunc callback function called when a plugin with old plugin protocol is started.
```

Comment:
I'm not sure if "callback" is best here. If it is a once every start thing (and when restarted?). The maybe InitFuncs or something?